source/adminguide/host_and_storage_tags.rst

```diff
@@ -49,6 +49,37 @@ To explain the behavior of host tags, some examples will be demonstrated with tw
     * Offering: h3
     None of the hosts have compatible tags and it will not be possible to deploy a VM with the offering. However, CloudStack ignores this behavior when a host is manually selected.
 
+.. _strict-host-tags:
+Strict Host Tags
+-----------------
+During certain operations, like changing compute offering, and starting or live
+migrating an instance to a specific host, CloudStack will ignore the host tags,
+which can lead to issues where a instance starts on an undesired host.
+
+To avoid a mismatch of tags, tags which must always match during any operation
+can be defined in `vm.strict.host.tags`. If the tags which doesn't match are
```

Comment:
explain this is global configuration, it doesn't say it's a global config 'vm.strict.host.tags'....